src/snowflake/snowpark/_internal/compiler/utils.py

```diff
@@ -439,7 +446,7 @@ def get_name(node: Optional[LogicalPlan]) -> str:  # pragma: no cover
         if isinstance(node, SnowflakePlan):
             name = f"{name} :: ({get_name(node.source_plan)})"
         elif isinstance(node, SelectSnowflakePlan):
-            name = f"{name} :: ({get_name(node.snowflake_plan.source_plan)})"
+            name = f"{name} :: ({get_name(node.snowflake_plan)}) :: ({get_name(node.snowflake_plan.source_plan)})"
```

Comment:
this change only affects plan_plotter for debugging. Does not affect production code